### PR TITLE
feat(web): Sýslumenn footer links to support page

### PR DESCRIPTION
**Pull request by RunarVestmann**
# Sýslumenn footer links to support page

## What

* Added a link to the Sýslumenn footer that links to the Sýslumenn support page
* Placed the change behind a feature flag

## Why

* This was a requested feature,

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/web/components/Organization/Wrapper/Themes/SyslumennTheme/SyslumennFooter.tsx [hunk 1]
```diff
@@ -22,14 +23,23 @@ interface FooterProps {
...
+  questionsAndAnswersText?: string
+  canWeHelpText?: string
 }
 
 export const SyslumennFooter: React.FC<FooterProps> = ({
   title,
   logo,
   footerItems,
+  questionsAndAnswersText = 'Spurningar og svör',
+  canWeHelpText = 'Getum við aðstoðað?',
```
Comment (kristofer-lp8f) [inline apps/web/components/Organization/Wrapper/Themes/SyslumennTheme/SyslumennFooter.tsx:35]: i18n?

**Review by kristofer-lp8f — APPROVED** [review]
LGTM, ~except for the i18n~ :+1:

Up to you how to handle the i18n.

File: apps/web/components/Organization/Wrapper/Themes/SyslumennTheme/SyslumennFooter.tsx (hunk 1 above)
Comment (RunarVestmann) [reply apps/web/components/Organization/Wrapper/Themes/SyslumennTheme/SyslumennFooter.tsx:35]: image
I am using translations, these are just default values if you do not provide the props

Comment (kristofer-lp8f) [reply apps/web/components/Organization/Wrapper/Themes/SyslumennTheme/SyslumennFooter.tsx:35]: Right, my bad. Good job :)

**Pull request merged**